### ISSUE TITLE
Unable to install openblas with vcpkg

### ISSUE THREAD

**Issue by DKGH** [microsoft/vcpkg#35071]
**Describe the bug**
I am using vcpkg in manifest mode to install mlpack. Below is the log:

```
C:\Users\user\source\repos\ml-trails>vcpkg install
Fetching registry information from https://github.com/microsoft/vcpkg (HEAD)...
Detecting compiler hash for triplet x64-windows...
The following packages will be built and installed:
  * armadillo:x64-windows -> 12.0.1 -- C:\Users\user\AppData\Local\vcpkg\registries\git-trees\8e222a2972a96da2fbb07a76585574e9512b107f
  * blas:x64-windows -> 2023-03-25 -- C:\Users\user\AppData\Local\vcpkg\registries\git-trees\0fe7611c8bb3098083440074a6c59f35bfee4f99
  * cereal:x64-windows -> 1.3.2#1 -- C:\Users\user\AppData\Local\vcpkg\registries\git-trees\075869fcf5302c6dd11d564286d0dfa1d2d4d7a1
  * ensmallen:x64-windows -> 2.19.0 -- C:\Users\user\AppData\Local\vcpkg\registries\git-trees\00de3f8f709bd51bba23d6aa8f01636a1e99f298
  * lapack:x64-windows -> 2022-02-22#2 -- C:\Users\user\AppData\Local\vcpkg\registries\git-trees\7c892b58c70ebc7d83444c4e727272cad40e28c2
  * lapack-reference[blas-select,core,noblas]:x64-windows -> 3.11.0#1 -- C:\Users\user\AppData\Local\vcpkg\registries\git-trees\d71a74d50d88f3cf337d7cefccadcb192e7d12ab
    mlpack[core,openmp]:x64-windows -> 4.0.1 -- C:\Users\user\AppData\Local\vcpkg\registries\git-trees\274dd7a42860f7c70f7b94d5f8e3ad9083fa6e0d
  * openblas:x64-windows -> 0.3.23 -- C:\Users\user\AppData\Local\vcpkg\registries\git-trees\716c00140c032ec22773130b006af542e011f00e
  * stb:x64-windows -> 2023-04-11#1 -- C:\Users\user\AppData\Local\vcpkg\registries\git-trees\acd9c2bf96a3e32fbf629b1bfd6cba4827761069
  * vcpkg-cmake:x64-windows -> 2023-05-04 -- C:\Users\user\AppData\Local\vcpkg\registries\git-trees\88a7058fc7fa73a9c4c99cfcae9d79e2abf87a5a
  * vcpkg-cmake-config:x64-windows -> 2022-02-06#1 -- C:\Users\user\AppData\Local\vcpkg\registries\git-trees\8d54cc4f487d51b655abec5f9c9c3f86ca83311f
  * vcpkg-gfortran:x64-windows -> 3#3 -- C:\Users\user\AppData\Local\vcpkg\registries\git-trees\1af46bd54a9156a9f24fc7c8715488699522b382
Additional packages (*) will be modified to complete this operation.
Restored 2 package(s) from C:\Users\user\AppData\Local\vcpkg\archives in 50 ms. Use --debug to see more details.
Installing 1/12 vcpkg-cmake-config:x64-windows...
Elapsed time to handle vcpkg-cmake-config:x64-windows: 10.5 ms
Installing 2/12 vcpkg-cmake:x64-windows...
Elapsed time to handle vcpkg-cmake:x64-windows: 12.6 ms
Installing 3/12 stb:x64-windows...
Building stb:x64-windows...
-- Installing port from location: C:\Users\user\AppData\Local\vcpkg\registries\git-trees\acd9c2bf96a3e32fbf629b1bfd6cba4827761069
-- Using cached nothings-stb-5736b15f7ea0ffb08dd38af21067c314d6a3aae9.tar.gz.
-- Extracting source C:/Users/user/AppData/Local/vcpkg/downloads/nothings-stb-5736b15f7ea0ffb08dd38af21067c314d6a3aae9.tar.gz
-- Using source at C:/Users/user/source/repos/ml-trails/vcpkg_installed/vcpkg/blds/stb/src/14d6a3aae9-dc9b68ccde.clean
-- Installing: C:/Users/user/source/repos/ml-trails/vcpkg_installed/vcpkg/pkgs/stb_x64-windows/share/stb/FindStb.cmake
-- Installing: C:/Users/user/source/repos/ml-trails/vcpkg_installed/vcpkg/pkgs/stb_x64-windows/share/stb/vcpkg-cmake-wrapper.cmake
-- Installing: C:/Users/user/source/repos/ml-trails/vcpkg_installed/vcpkg/pkgs/stb_x64-windows/share/stb/usage
-- Installing: C:/Users/user/source/repos/ml-trails/vcpkg_installed/vcpkg/pkgs/stb_x64-windows/share/stb/copyright
-- Performing post-build validation
Stored binaries in 1 destinations in 108 ms.
Elapsed time to handle stb:x64-windows: 775 ms
Installing 4/12 openblas:x64-windows...
Building openblas:x64-windows...
-- Installing port from location: C:\Users\user\AppData\Local\vcpkg\registries\git-trees\716c00140c032ec22773130b006af542e011f00e
-- Using cached xianyi-OpenBLAS-394a9fbafe9010b76a2615c562204277a956eb52.tar.gz.
-- Extracting source C:/Users/user/AppData/Local/vcpkg/downloads/xianyi-OpenBLAS-394a9fbafe9010b76a2615c562204277a956eb52.tar.gz
-- Applying patch uwp.patch
-- Applying patch fix-redefinition-function.patch
-- Applying patch fix-uwp-build.patch
-- Applying patch install-tools.patch
-- Using source at C:/Users/user/source/repos/ml-trails/vcpkg_installed/vcpkg/blds/openblas/src/77a956eb52-e24e9a6e76.clean
-- Found external ninja('1.11.0').
-- Configuring x64-windows
```

And it gets stuck there. vcpkg contents:
```
"builtin-baseline": "638b1588be3a265a9c7ad5b212cef72a1cad336a",
"dependencies": [
  {
    "name": "mlpack",
    "default-features": true,
    "features": [ "openmp" ]
  }
]
```

**Environment**
- OS: Windows
- Compiler: MSVC. I also have Intel's Fortran compiler.

**To Reproduce**
1. Attempt installing mlpack in manifest mode with Windows 10 SDK. Intel's Fortran compiler also needs to be installed.

**Expected behavior**
The installation should finish successfully.

**Comment by DKGH** [comment]
vcpkg.json:

```
"builtin-baseline": "638b1588be3a265a9c7ad5b212cef72a1cad336a",
"dependencies": [
  {
    "name": "mlpack",
    "default-features": true,
    "features": [ "openmp" ]
  }
]
```

**Comment by Adela0814** [comment]
I'm not able to repro this issue.

**Comment by DKGH** [comment]
I uninstalled Intel compilers (and the whole Intel oneAPI) and reinstalled Visual Studio Community edition (v17.8.0). Looks like Intel was not the problem because I am still getting stuck at installing openblas. I also launched vcpkg with --debug option. Still, the last message I get is `Configuring x64-windows` and then nothing. In fact, I have to completely restart my PC to free up some of the files that vcpkg was writing to. The logs with --debug option is attached. Is there a way to make the logs more verbose?
[vcpkg_error.log](https://github.com/microsoft/vcpkg/files/13400623/vcpkg_error.log)
